### DISCUSSION TITLE
Supersede request behavior of llama-server

### DISCUSSION THREAD

**Discussion by 0x0539** [ggml-org/llama.cpp#10925] [Q&A]
I plan to run llama-server locally with the cuda backend to service autocomplete requests. I only care about the last-sent-request.

Is there a way to configure llama-server to "cancel all" requests when a new one is received?

Maybe I could set drop head queue with max size 1. Super best would be to actually cancel cuda before decoding finishes.

**Comment by ggerganov** [comment]
Unfortunately, there is no cancellation support implemented yet.
